src/seedu/addressbook/data/AddressBook.java

```diff
@@ -1,9 +1,6 @@
 package seedu.addressbook.data;
 
-import java.util.HashMap;
-import java.util.HashSet;
-import java.util.Map;
-import java.util.Set;
+import java.util.*;
```

Comment:
Importing whole java package (`java.util` in this case) is not a recommended practice due to redundant overhead operation. Intellij can help you auto-import java classes which are missing.